container/dockershell.sh

```diff
@@ -49,39 +84,74 @@ while [ -n "$1" ]; do
         shift 2
         ;;
     *)
-        echo "Unknown option:$1"
+        printf "Unknown option:$1"
         usage >&2
         exit 1
     esac
 done
 
 if [ -z ${EVENT_USER} ]; then
-    echo "Error: Please provide the Watson Studio Local user name with --user flag"
+    printf "Error: Please provide the cluster user name with --user flag\n"
```

Comment:
since desktop is not really a cluster, maybe just call this Db2 Event Store instance ?